src/components/CommandPalette.tsx

```diff
@@ -197,7 +204,22 @@ const CommandRow: FC<{
           {isTouch && (
             <GestureDiagram
               color={disabled ? token('colors.gray') : undefined}
-              highlight={!disabled ? gestureInProgress.length : undefined}
+              highlight={
+                !disabled
+                  ? shortcut.id === 'help'
+                    ? // For help command, only highlight the matching portion at the end
+                      (() => {
+                        const helpGesture = gestureString(shortcut)
+                        const subsequences = Array.from({ length: helpGesture.length }, (_, i) =>
```

Comment:
This is acceptable, but note that there is a way to do this more efficiently by iterating from the end of the gesture. You shouldn't have to generate all the subsequences.